### DISCUSSION TITLE
Welcome to Bing-Chat-Saver Discussions!

### DISCUSSION THREAD

**Discussion by gantrol** [gantrol/Bing-Chat-Saver#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by llkj11** [comment]
For some reason your Bing Chat Saver extension no longer allows for the personality personalization for me on Bing. When disabled it shows up again.

**Reply by gantrol** [reply]
Thanks for feedback. Personal settings are stored and synchronized using the chrome.storage.sync interface, and disabling the plugin should not affect it. I will try to reproduce this issue, and if you could provide more information such as the browser version and plugin version, it would be helpful for troubleshooting.

**Reply by llkj11** [reply]
No problem! I am on the Microsoft Edge Dev browser Version 111.0.1661.39 (Official build) dev (64-bit). I am on Windows 11 pro running the new h2h2 build. I noticed the issue a few days ago and restarted my PC and reinstall my browser. Initially I thought they threw me on another build of Bing Chat for whatever reason, but someone on Reddit told me to disable your extension and it worked right after.
